src/locales.json

```diff
@@ -478,5 +478,35 @@
     "delete-post": "Futa posti",
     "hint-post": "Dokeza",
     "private-channel-hint": "🆘 Ukitaka kuongeza watumiaji kwenye chaneli binafsi, wanahitaji kuwa wamesaini ndani ya mojibrag teyari. Kama bado, waambie wajisajili kwanza, na uwajumuishe baada ya hapo!"
+  },
+  "ta": {
+    "flag": "TA",
```

Comment:
@notwaldorf that should work. It might potentially lead to confusion when someone else from India wants to add [their own language they use in India as well](https://en.wikipedia.org/wiki/Languages_of_India#Scheduled_languages), which will also end up in the same country flag emoji.